LibreNMS/Device/YamlDiscovery.php

```diff
@@ -100,7 +102,7 @@ public static function discover(OS $os, $class, $yaml_data)
                     }
 
                     foreach ($data as $name => $value) {
-                        if ($name == '$oid' || $name == 'skip_values') {
+                        if (in_array($name, ['oid', 'skip_values', 'snmp_flags'])) {
```

Comment:
Hi @murrant 
Found that this PR added "snmp_flags" in this row, allowing it to contain a variable that will be replaced. Did you do it with a precise usecase in mind ?

Reply:
To long ago, the change was probably intentional